winter-2020/paramedics-of-waterloo-adr.md

```diff
@@ -0,0 +1,75 @@
+# Region of Waterloo Paramedic Services (ROWPS)
+
+- Project Lead: Peter Song (@petorsong)
+- Developers:
+	- Aaron Abraham (@aaronabraham311)
+	- Carelynn Tsai (@carelynntsai)
+	- Daniel Chen (@danielchen115)
+	- Jason Guo (@JasonYG)
+	- Jeffrey Kam (@lazypanda10117)
+	- Ricky Mao (@rickrm)
+- Product Manager: Rishabh Patni
+- Designers:
+	- Charmaine Wang
+	- Gracie Xia
+	- Joe Li
+
+## Links to relevant documentation
+
+-	[Statement of work](https://docs.google.com/document/d/11omQuizb0RjQG6-ynfOJr6mn_birUFtSVUMyLnqUAkY/edit?usp=sharing)
+- [Initial technical scoping](https://www.notion.so/uwblueprintexecs/2019-Technical-Doc-6e1dfdf27536441cbce8234819384ba6)
+- [Notion workspace](https://www.notion.so/uwblueprintexecs/Region-of-Waterloo-Paramedic-Services-396794328ff14336a1c759e84e83b370)
+
+## Summary of Project
+
+- ROWPS is looking for a solution to coordinate personnel and keep track of patients during events with an unusually high number of patients (e.g. Homecoming, St. Patrick's Day @ Ezra Ave.)
+- We will build and deliver an iPad application to be used by paramedics on the scene to manage patients and a web application to be used for monitoring
+- This project is scoped to be completed in two terms (W20, S20)
+  - The first term will focus on developing the backend and beginning on the web frontend
+  - The second term will focus on completing the web and mobile clients
+
+## Existing Solutions (either in BP or externally)
+
+-	No existing technical solutions found
+
+## Technical Design

```

Comment:
Has any thought been put into automating deployment or will their IT team need to go into every EC2 instance and run a shell command to spin up each service?

Reply:
because we're using a docker setup, it should be easy to automate deployment - we will look into specifics